test/packages-src/bonjour/manifest.mf

```diff
@@ -38,7 +38,15 @@
       "executable": "{{.PackageDir}}/bonjour.{{if eq .Os \"windows\"}}bat{{else}}sh{{end}}",
       "args": [],
       "validArgsCmd": [ "{{.PackageDir}}/auto-complete.{{if eq .Os \"windows\"}}bat{{else}}sh{{end}}" ]
+    },
+    {
+      "name": "saybonjour2",
```

Comment:
Is this new command really usef?

Reply:
You are right, this test command should be deleted. It is intended to cover a different use case, which is removed from this patch.